__tests__/RequestVerifier.spec.ts

```diff
@@ -0,0 +1,139 @@
+            test("throws 'Error with request header, signatures are not present'", () => {
+                const tolerance: number = 5 * 60
+                const requestBody: string = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}"
+                const signingSecret: string = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+                const requestHeader: string = "t=1679944186,";
+                expect(() => {instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance)}).toThrow("Error with request header, signatures are not present");
+            }) 
+        })
+        describe("timestamp is not present", () => {
+            test("throws 'Error with request header, timestamp is not present'", () => {
+                const tolerance: number = 5 * 60
+                const requestBody: string = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}"
+                const signingSecret: string = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+                const requestHeader: string = "v1=c3957749baf61df4b1506802579cc69a74c77a1ae21447b930e5a704f9ec4120,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8";
+                expect(() => {instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance)}).toThrow("Error with request header, timestamp is not present");
+            }) 
+        })
+        describe("Request header is empty or null", () => {
+            test("throws 'Error with request header, Request header is empty'", () => {
+                const tolerance: number = 5 * 60
+                const requestBody: string = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}"
+                const signingSecret: string = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+                const requestHeader: string = "" 
+                expect(() => {instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance)}).toThrow("Error with request header, Request header is empty");
+            }) 
+        })  
+    })
+    describe("#checkSigningSecret", () => {
+        describe("Signing secret is empty or null", () => {
```

Comment:
Break this out into multiple cases. e.g.

- Signing secret is empty
- Signing secret is null

Reply:
Same as above comment for requestBody splitting test cases comment